tests/sdk/test_schemas.py

```diff
@@ -38,3 +40,21 @@ def test_datasample_spec_paths_set_to_none():
 def test_datasample_spec_path_set_to_none():
     with pytest.raises(ValueError):
         DataSampleSpec(path=None, data_manager_keys=[str(uuid.uuid4())])
+
+
+def test_dataset_spec_no_description(tmpdir):
+
+    opener_path = tmpdir / "fake_opener.py"
+    with open(opener_path, "w") as f:
+        f.write("print('I'm opening your data')")
```

Comment:
I'm not sure whether we can mock this or not but it is usually considered as a good practice to limit fs interactions in unit tests

Reply:
Even in the pytest tmpdir ? But your right, I don't need to write the file, just to pass the path, good catch :)